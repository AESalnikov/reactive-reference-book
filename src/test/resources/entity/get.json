{
  "firstName": "Федор",
  "lastName": "Федоров",
  "middleName": "Федорович",
  "positionHeld": "Администратор",
  "email": "fedorov-ff@test.ru",
  "phoneNumber": "88000000002"
}
