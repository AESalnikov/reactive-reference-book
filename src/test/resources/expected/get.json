{
  "id": "00000000-0000-0000-0000-000000000002",
  "firstName": "Федор",
  "lastName": "Федоров",
  "middleName": "Федорович",
  "positionHeld": "Администратор",
  "email": "fedorov-ff@test.ru",
  "phoneNumber": "88000000002"
}
